src/cli/main.py

```diff
@@ -226,62 +284,51 @@ def get_job(ctx: click.Context, name: str) -> None:
     default=["westeurope"],
     help="Regions where the test resources will be provisioned.",
 )
+@click.option(
+    "--concurrency",
+    "-c",
+    default=0,
+    type=click.IntRange(min=0, max=4, clamp=False),
+    help="The number of concurrent tests to run in parallel",
```

Comment:
`concurrent` and `in parallel` are redundant.

Also, the validity of `0` here is confusing. The lower bound should be `1` (one test is running at a time). I realize, however, that you are simply passing this value through to the API.

Reply:
Yes, this is confusing. The default when nothing is passed to the API is 0. I don't know what 0 is for and what is the difference between 0 an 1.